src/main/java/edu/indiana/dlib/amppd/controller/WorkflowController.java

```diff
@@ -26,8 +28,19 @@
 	 * @return
 	 */
 	@GetMapping("/workflows")
-	public GalaxyWorkflow[] getWorkflows() {		
-		return galaxyApiService.getWorkflows();
+	public List<Workflow> getWorkflows() {		
+		List<Workflow> workflows = galaxyApiService.getInstance().getWorkflowsClient().getWorkflows();
+		log.info("Retrieved " + workflows.size() + " current workflows in Galaxy: " + workflows);
+		return workflows;
 	}
 
+//	/**
+//	 * Retrieve all workflows from Galaxy through its REST API.
+//	 * @return
+//	 */
+//	@GetMapping("/workflows")
```

Comment:
Remove commented code, that is not needed, we can always go back in git history to get the code again, if needed.